modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectDefinitionLocalServiceImpl.java

```diff
@@ -1648,6 +1653,22 @@ private void _validateEnableObjectEntryHistory(
 		}
 	}
 
+	private void _validateExternalReferenceCode(
+			String externalReferenceCode, String objectDefinitionName,
```

Comment:
Could you sort the parameters? I think objectDefinitionName comes after modifiable